symphony-bdk-legacy/symphony-bdk-bot-sdk-java/src/test/java/com/symphony/bdk/bot/sdk/extapp/logging/LogControllerTest.java

```diff
@@ -0,0 +1,111 @@
+package com.symphony.bdk.bot.sdk.extapp.logging;
+
+import com.symphony.bdk.bot.sdk.symphony.ConfigClient;
+
+import lombok.SneakyThrows;
+import org.junit.*;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+public class LogControllerTest {
+  private final String FILE_NAME = "src/test/resources/logs/test.log";
+  private ConfigClient configClient;
+
+  @SneakyThrows
+  @Before
+  public void init() {
+    try {
+      Files.createFile(Paths.get(FILE_NAME));
```

Comment:
Could you consider using [File#createTempFile](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#createTempFile(java.nio.file.Path,%20java.lang.String,%20java.lang.String,%20java.nio.file.attribute.FileAttribute...)) instead?